common/src/chain/config/builder.rs

```diff
@@ -236,6 +268,30 @@ impl Builder {
         let target_block_spacing = super::DEFAULT_TARGET_BLOCK_SPACING;
         let consensus_upgrades = chain_type.default_consensus_upgrades(target_block_spacing);
 
+        let nft_issuance_fee = if chain_type == ChainType::Testnet {
+            super::TESTNET_NFT_ISSUANCE_FEE
+        } else {
+            super::MAINNET_NFT_ISSUANCE_FEE
+        };
+
+        let token_supply_change_fee = if chain_type == ChainType::Testnet {
+            super::TESTNET_TOKEN_SUPPLY_CHANGE_FEE
+        } else {
+            super::MAINNET_TOKEN_SUPPLY_CHANGE_FEE
+        };
+
+        let token_freeze_fee = if chain_type == ChainType::Testnet {
+            super::TESTNET_TOKEN_FREEZE_FEE
+        } else {
+            super::MAINNET_TOKEN_FREEZE_FEE
+        };
+
+        let token_change_authority_fee = if chain_type == ChainType::Testnet {
+            super::TESTNET_TOKEN_CHANGE_AUTHORITY_FEE
+        } else {
+            super::MAINNET_TOKEN_CHANGE_AUTHORITY_FEE
+        };
```

Comment:
Minor: May be a bit nicer to collect these into a sort of table instead of a cascade of if-else.

Reply:
I reworked this in #1472